tools/tm-signer-harness/main.go

```diff
@@ -59,6 +60,14 @@ Use "tm-signer-harness help <command>" for more information about that command.`
 		fmt.Println("")
 	}
 
+	hd, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Println("The UserHomeDir is not defined, setting the default TM Home PATH to \"~/.tendermint\"")
+		defaultTMHome = "~/.tendermint"
```

Comment:
It looks like this is just expanded with the `user.Current()` call here https://github.com/tendermint/tendermint/blob/21a3bbda3fd8d272cc7fee888b4339a669cbcaad/tools/tm-signer-harness/main.go#L110
https://github.com/tendermint/tendermint/blob/21a3bbda3fd8d272cc7fee888b4339a669cbcaad/tools/tm-signer-harness/internal/utils.go#L13

Not sure I see what benefit this brings?

Reply:
didn't notice there is a util function for it. just updated.